apps/www/src/content/docs/components/data-table.md

```diff
@@ -55,6 +55,18 @@ npm install @tanstack/vue-table
 
 <ComponentPreview name="DataTableColumnPinningDemo" />
 
+### Reactive Table
+
+A reactive table was added in `v8.20.0` of the TanStack Table. You can see the [docs](https://tanstack.com/table/latest/docs/framework/vue/guide/table-state#using-reactive-data) for more information. We added an example where we are randomizing `status` column. One main point is that you need to mutate **full** data, as it is a `shallowRef` object.
+
+> __*⚠️ `shallowRef` is used under the hood for performance reasons, meaning that the data is not deeply reactive, only the `.value` is. To update the data you have to mutate the data directly.*__
+
+Relative PR: [Tanstack/table #5687](https://github.com/TanStack/table/pull/5687#issuecomment-2281067245)
+
+If you want to mutate `props.data`, you should use `defineModule`.
```

Comment:
`defineModel`?

Reply:
@sadeghbarati thanks! 😁 

<img width="555" alt="image" src="https://github.com/user-attachments/assets/89d12cc9-df87-49cd-b588-e0c4376e9ff5">

Also added link to official vue docs 🙌

Reply:
Resolved in [`6d4ed04` (#744)](https://github.com/radix-vue/shadcn-vue/pull/744/commits/6d4ed04cd21ea05d222dd3a2b489bdff9994c719)